testsuite/forge-cli/src/main.rs

```diff
@@ -421,19 +430,35 @@ fn main() -> Result<()> {
                 }
                 Ok(())
             },
-            OperatorCommand::Resize(resize) => {
-                runtime.block_on(install_testnet_resources(
-                    resize.namespace,
-                    resize.num_validators,
-                    resize.num_fullnodes,
-                    resize.validator_image_tag,
-                    resize.testnet_image_tag,
-                    resize.move_modules_dir,
-                    !resize.connect_directly,
-                    resize.enable_haproxy,
-                    None,
+            OperatorCommand::Create(create) => {
+                let kube_client = runtime.block_on(create_k8s_client())?;
+                let era = generate_new_era();
+                let config: Value = serde_json::from_value(json!({
+                    "profile": DEFAULT_FORGE_DEPLOYER_PROFILE.to_string(),
+                    "era": era,
+                    "namespace": create.namespace.clone(),
+                }))?;
+                let testnet_deployer = ForgeDeployerManager::new(
+                    kube_client.clone(),
+                    create.namespace.clone(),
+                    FORGE_TESTNET_DEPLOYER_DOCKER_IMAGE_REPO.to_string(),
                     None,
-                ))?;
+                    config.clone(),
+                );
+                // NOTE: this is generally not going to run from within the cluster, do not perform any operations
+                // that might require internal DNS resolution to work, such as txn emission directly against the node service IPs.
+                runtime.block_on(testnet_deployer.start())?;
+                if create.enable_indexer {
+                    let indexer_deployer = ForgeDeployerManager::new(
+                        kube_client,
+                        create.namespace,
+                        FORGE_INDEXER_DEPLOYER_DOCKER_IMAGE_REPO.to_string(),
+                        None,
+                        config,
```

Comment:
The only difference between the two deployers here seems to be the values

Instead of passing the values to the deployer (which doesn need to hold state) you could instead init them with a client / namespce and then pass values to start